src/renderer/components/message/MessageList.tsx

```diff
@@ -250,6 +226,52 @@ export const MessageListInner = React.memo(
   }
 )
 
+function EmptyChatMessage({ chat: chatStoreState }: { chat: ChatStoreState }) {
+  const tx = useTranslationFunction()
+
+  const [chat, setChat] = useState<FullChat>(chatStoreState)
+  useEffect(() => {
+    const refresh = (_ev: any, chatId: number) => {
+      if (chatId == chat.id) {
+        DeltaBackend.call('chatList.getFullChatById', chat.id).then(
+          fullChat => {
+            setChat(fullChat)
+          }
```

Comment:
Shouldn't this get handled by the chat store?

Reply:
probably, but I didn't want to deal with the chatstore. If you want to move it there feel free to do so.

Reply:
looks like you're right this gets already handled by the chat store, I'll change it to use the chat-store instead.